feat(serverless-contracts): add an optional arg in getHandler to disable input/output validation (#526)
swarmion/swarmion: user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/2-eventbridge.md

:::caution
This handler also provides a payload validation that will throw an error if there is a mismatch with the `payloadSchema`. This ensure that invalid events will not be mistakenly taken into account. However, be sure to set up an invalid events failure flow, for example with a [Lambda `onFailure` destination](https://www.serverless.com/blog/lambda-destinations/).

> If you still wish to disable this behavior, you can use a second options argument int the `getHandler` feature:

Reviewer: ```suggestion
If you still wish to disable this behavior, you can use the optional second argument in the `getHandler` feature:
```
